<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Bernd Stramm</author>
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Text</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../permute.qrc">
    <normaloff>:/deeptrim.png</normaloff>:/deeptrim.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="searchLabel">
       <property name="text">
        <string>Search For</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="needleEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="replaceLabel">
       <property name="text">
        <string>Replace with</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="replaceEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkCase">
       <property name="text">
        <string>Case Sensitive</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="checkWord">
       <property name="text">
        <string>Word</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkWrap">
       <property name="text">
        <string>Wrap around Bottom</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="checkRegular">
       <property name="text">
        <string>Regular Expression</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="cancelButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>10</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="replaceAllButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>5</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Replace All</string>
       </property>
       <attribute name="buttonGroup">
        <string>replaceGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replaceButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>5</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Replace</string>
       </property>
       <attribute name="buttonGroup">
        <string>replaceGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="searchButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>10</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Find</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../permute.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="replaceGroup"/>
 </buttongroups>
</ui>
